ast/src/statements/statement.rs

```diff
@@ -24,13 +24,22 @@ use std::fmt;
 /// Program statement that defines some action (or expression) to be carried out.
 #[derive(Clone, PartialEq, Eq, Serialize, Deserialize, Debug)]
 pub enum Statement {
+    /// A return statement `return expr;`.
     Return(ReturnStatement),
+    /// A binding or set of bindings / variables to declare.
     Definition(DefinitionStatement),
+    /// An assignment statement.
     Assign(Box<AssignStatement>),
+    /// An `if` statement.
     Conditional(ConditionalStatement),
+    /// A `for` statement.
     Iteration(Box<IterationStatement>),
+    /// A console logging statement.
```

Comment:
I would just say 'A console statement', because it may be a logging one or another kind.